### PR TITLE
Display all user types and org types on admin management UI

### PR DESCRIPTION
**Pull request by lunny**
Follow #24026 

图片

### CODE REVIEW

**Comment by puni9869** [comment]
image
Can these label be a lil small in size  they will take much space once username is big. Thats why.

**Comment by lunny** [comment]
> image
> 
>     1. Can these label be a lil small in size  they will take much space once username is big. Thats why.
> 
>     2. What is the difference in `admin.reserved` and `admin` type.

Changed to small size label.

File: templates/admin/user/list.tmpl [hunk 1]
```diff
@@ -84,7 +84,13 @@
 							<td>
 								<a href="{{$.Link}}/{{.ID}}">{{.Name}}</a>
 								{{if .IsAdmin}}
-									<span class="ui basic label">{{$.locale.Tr "admin.users.admin"}}</span>
+									<span class="ui mini label">{{$.locale.Tr "admin.users.admin"}}</span>
+								{{else if eq 2 .Type}}
+									<span class="ui mini label">{{$.locale.Tr "admin.users.reserved"}}</span>
+								{{else if eq 4 .Type}}
+									<span class="ui mini label">{{$.locale.Tr "admin.users.bot"}}</span>
+								{{else if eq 5 .Type}}
```
Comment (delvh) [inline templates/admin/user/list.tmpl]:
```suggestion
								{{else if eq 2 .Type}}{{/* Reserved user */}}
									<span class="ui mini label">{{$.locale.Tr "admin.users.reserved"}}</span>
								{{else if eq 4 .Type}}{{/* Bot "user" */}}
									<span class="ui mini label">{{$.locale.Tr "admin.users.bot"}}</span>
								{{else if eq 5 .Type}}{{/* Remote user */}}
```

File: templates/admin/org/list.tmpl [hunk 2]
```diff
@@ -37,6 +37,9 @@
 								{{if .Visibility.IsPrivate}}
 									<span class="text gold">{{svg "octicon-lock"}}</span>
 								{{end}}
+								{{if eq .Type 3}}
```
Comment (delvh) [inline templates/admin/org/list.tmpl]:
```suggestion
								{{if eq .Type 3}}{{/* Reserved organization */}}
```

**Review by delvh — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Pull request merged**